portfolio/src/main/webapp/index.html

```diff
@@ -126,9 +126,8 @@ <h1 class="title">
               login in different
               services with the same account. Reaching more than <strong>10 organizations</strong>, and almost
               <strong>10,000 users</strong> across platforms</p>
-            <p class="is-size-7 mt-2">Also, I created a document certification and signature protocol based on <strong>Ethereum
-                Blockchain and the Elliptic Curve Digital Signature</strong> algorithm which allows to upload a certify many type
-              of documents</p>
+            <p class="is-size-7 mt-2">Also, I created a document certification and signature protocol based on
+              <strong>Ethereum Blockchain and the Elliptic Curve Digital Signature</strong> algorithm which allows to upload a certify many type of documents</p>
```

Comment:
I think there might be a word missing or mistyped here? I'm not sure what it is though. Maybe "allows **users** to upload **and** certify many types of documents"?

Reply:
OMG! Yes, it is an error. I solved it, thank you!